BookAPI/Program.cs

```diff
@@ -10,19 +8,23 @@ static void Main(string[] args)
         {
             Console.WriteLine("Book Exchange");
             BookModel newBook = new BookModel();
+            UserRepository userRepository = new UserRepository();
             string bookName, bookOwner;
             bool bookAvailability = false;
+
             Console.WriteLine("Please Enter Book name");
             bookName = Console.ReadLine();
             BookRepository.Instance.IsValidBookName(bookName);
             Console.WriteLine("Please Enter Book Owner name");
             bookOwner = Console.ReadLine();
             BookRepository.Instance.IsValidBookOwnerName(bookOwner);
+            userRepository.IsBookOwnerExistsAlready(bookOwner);
             newBook.BookName = bookName;
             newBook.OwnerName = bookOwner;
             newBook.AvailabilityStatus = bookAvailability;
             int result = BookRepository.Instance.AddNewBook(newBook);
             Console.WriteLine(result);
+
```

Comment:
Next steup would be removing all validations here, just take input and call AddNewBook
Program.cs is just a way to call the method.

Reply:
Removed all validations and moved to respective repository classes